design-documents/webhooks.md

```diff
@@ -0,0 +1,160 @@
+## Webhooks
```

Comment:
I am trying to understand the benefits of webhooks over message queue for us, considering the cost of implementation and support. 

Are there any use cases not covered with existing message queue framework?

Reply:
@paliarush AFAIK there isn't a way to easily use message queue to call external http endpoints?